charts/prometheus/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: prometheus
 appVersion: 2.31.1
-version: 15.0.4
+version: 15.0.5
```

imho this should be a minor version to as minor version of subchart is changed.
```suggestion
version: 15.1.0
```

---

Got it, I changed it to a minor version instead.